starter_code/queries.md

```diff
@@ -5,42 +5,93 @@
 ### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
 
 <!-- Your Code Goes Here -->
+{name: "Babelgum"}
```

Comment:
Yes, and if you want to retrieve only the name field, you can add this in the projection : 
```
{name: 1, _id: 0}
```